src/main/java/persistence/Persistence.java

```diff
@@ -20,4 +20,8 @@ public abstract class Persistence {
     DataSource dataSource = new HikariDataSource(hikariConfig);
     jdbi = Jdbi.create(dataSource);
   }
+
+  Persistence(Jdbi jdbi) {
+    this.jdbi = jdbi;
+  }
```

Comment:
to jest próba, która nie wiem co am dać ale chuj zobaczymy robię to pierwszy raz i jest zjebane na serio